src/bvals/comms/bnd_info.hpp

```diff
@@ -78,11 +71,68 @@ struct BndInfo {
                                  CommBuffer<buf_pool_t<Real>::owner_t> *buf);
 };
 
+struct ProResInfo {
```

Comment:
I *think* it makes sense to split things out this way. At least I don't object.

Reply:
The rational was that it the boundary communication information is unnecessary when doing prolongation and restriction in one during remeshing. Additionally, we may want to not build `ProResInfo` objects for regions that don't require prolongation/restriction at some point to minimize the storage here, since at least as written there are a bunch of `IdxRange`s in `ProResInfo`

Reply:
Also for GMG, I have to pair different `BndInfo` creators and `ProResInfo` creators than for normal boundary communication.

Reply:
:+1: that makes sense.